client/examples/tutorial.rs

```diff
@@ -185,3 +189,56 @@ fn asset_registration_test(config: &Configuration) -> Result<(), Error> {
     // Finish the test successfully
     Ok(())
 }
+
+fn asset_minting_test() -> Result<(), Error> {
```

Comment:
```suggestion
fn asset_minting_test() {
```
you're never returning error in the test

Reply:
or you could use `?` instead of `expect` which I don't think is a good idea in this case

Reply:
Thanks!